frontend/src/components/TaskTableUnrated.tsx

```diff
@@ -85,11 +85,13 @@ const TableUnratedTaskRow: TableRow<Task> = ({ item: task, style }) => {
     if (type === RoleType.Business) {
       setMissingRatings(
         userInfo?.representativeFor?.filter(
-          (customer) => !ratedByCustomer(customer, userInfo)(task),
+          (customer) =>
+            !ratedByCustomer(customer, userInfo)(task) &&
+            roadmapId === customer.roadmapId,
```

Comment:
```suggestion
            roadmapId === customer.roadmapId &&
            !ratedByCustomer(customer, userInfo)(task),
```

I'd put these checks in the other order, since the other is a simple value comparison